### PR TITLE
fixed update to the state when items are rearanged in a dropzone box ,

### PR DESCRIPTION
**Pull request by el0911**
fixed update to the state when items are rearranged in a dropzone, the issue was that the state was not  holding the current order for the canvas , so if u called getStorableState() after you've rearranged items  in a dropzone it doesn't give the current order, you can test this in the live example when viewing preview

### CODE REVIEW

File: src/core/state.js [hunk 1]
```diff
@@ -64,6 +64,8 @@ class State {
 
             // no new fields
             if (!fieldsToBeAdded.length) {
+              matchedParentCanvas.fields = updatedFields;
```
Comment (keermanish) [inline src/core/state.js]: Can you provide more insight on this line of code, as I could see reassignment of same variable at next line.

**Comment by keermanish** [comment]
Issue is fixed in latest 1.1.31. Closing this PR.

**Comment by keermanish** [comment]
Thanks for pointing it out. :)

**Comment by el0911** [comment]
No P , would like to talk to you about something would u be available
anytime

**Pull request closed without merging**